planner/core/planbuilder.go

```diff
@@ -2017,6 +2027,9 @@ func buildShowSchema(s *ast.ShowStmt, isView bool) (schema *expression.Schema) {
 	case ast.ShowBindings:
 		names = []string{"Original_sql", "Bind_sql", "Default_db", "Status", "Create_time", "Update_time", "Charset", "Collation"}
 		ftypes = []byte{mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeDatetime, mysql.TypeDatetime, mysql.TypeVarchar, mysql.TypeVarchar}
+	case ast.ShowAnalyzeStatus:
+		names = []string{"Table_schema", "Table_name", "Partition_name", "Job_info", "Row_count", "Start_time", "State"}
```

Comment:
It would be nice if we can distinguish auto analyze tasks with manual analyze tasks.

Reply:
Put it in the `job info`, see https://github.com/pingcap/tidb/pull/10172/files#diff-badfcd30d7596a08cd207b8e6ae778e6R1471